src/messages/MNVMeofGwMap.h

```diff
@@ -20,8 +20,8 @@
 
 class MNVMeofGwMap final : public Message {
 private:
-  static constexpr int VERSION = 1;
-
+  static int constexpr VERSION = 2;
```

Comment:
That's a pretty unusual order of declarations. I surely  would not have *fixed* 'constexpr int' into 'int constexpr'